### PR TITLE
renaming model Registration_Number to Registration, and adjusting API…

### PR DESCRIPTION
**Pull request by anissa-agahchen**
… to call /register

### CODE REVIEW

File: dcbr-api/api/models.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ class Operator(models.Model):
     
 
     def __str__(self):
-        return "Reg ID: \t %s %s , %s" % (self.registration_Number, self.last_name, self.first_name)
+        return "Reg ID: \t %s %s , %s" % (self.regNum, self.last_name, self.first_name)
```
Comment (esune) [inline dcbr-api/api/models.py]: We could leave the full spelling `registration_number` for consistency with the rest of the code, and clarity.

Comment (anissa-agahchen) [reply dcbr-api/api/models.py]: ok

**Comment by anissa-agahchen** [comment]
I changed a migration file, so I'm closing this PR and I'll open a new one shortly

**Comment by anissa-agahchen** [comment]
@esune can you take a look again?

**Pull request merged**